demos/consistent-focus.html

```diff
@@ -18,212 +19,219 @@
-            <calcite-action id="toggle-mode" text="Toggle mode" icon="brightness"></calcite-action>
-          </calcite-action-pad>
-          <calcite-tooltip placement="bottom" reference-element="toggle-mode" slot="content-end">Toggle mode</calcite-tooltip>
-      </calcite-navigation>
-
-      <calcite-shell-panel slot="panel-start" display-mode="float">
-        <calcite-action-bar slot="action-bar">
-          <calcite-action data-action-id="layer-effects" icon="effects" text="Layer effects"></calcite-action>
-        </calcite-action-bar>
-        <!-- Layer effects -->
-        <calcite-panel heading="Layer effects" height-scale="l" data-panel-id="layer-effects" closable closed>
-            <!-- Bloom effect -->
-            <calcite-block open heading="Bloom" description="Apply a neon-like glow">
-              <!-- Bloom enabled -->
-              <calcite-block-section id="bloom-effect" open text="Apply bloom effect" toggle-display="switch">
-              <!-- Intensity -->
-              <calcite-label for="bloom-intensity">Bloom intensity</calcite-label>
-              <calcite-slider id="bloom-intensity" value="1.5" min="0" max="10" step="0.5" label-ticks ticks="2"></calcite-slider>
-              <!-- Radius-->
-              <calcite-label for="bloom-radius">Bloom radius</calcite-label>
-              <calcite-slider id="bloom-radius" value="0.5" min="0" max="1" step="0.5" label-ticks ticks="0.25"></calcite-slider>
-              <!-- Threshold -->
-              <calcite-label for="bloom-threshold">Bloom threshold</calcite-label>
-              <calcite-slider id="bloom-threshold" value="0.1" min="0" max="0.5" step="0.1" label-ticks ticks="0.25"></calcite-slider>
-              </calcite-block-section>
-          </calcite-block>
-          <!-- Drop shadow effect -->
-          <calcite-block open heading="Drop shadow" description="Apply a drop shadow">
-            <!-- Shadow enabled -->
-            <calcite-block-section id="shadow-effect" text="Apply drop shadow effect" toggle-display="switch">
+<body class="calcite-mode-dark">
+  <calcite-shell content-behind>
+    <calcite-navigation slot="header" id="nav">
+      <calcite-navigation-logo icon="accessibility" slot="logo" heading="Consistent Focus"
+        description="Esri Developer Summit 2024"></calcite-navigation-logo>
+      <calcite-action-pad layout="horizontal" expand-disabled slot="content-end">
+        <calcite-action id="toggle-mode" text="Toggle mode" icon="brightness"></calcite-action>
+      </calcite-action-pad>
+      <calcite-tooltip placement="bottom" reference-element="toggle-mode" slot="content-end">Toggle
+        mode</calcite-tooltip>
+    </calcite-navigation>
+
+    <calcite-shell-panel slot="panel-start" display-mode="float">
+      <calcite-action-bar slot="action-bar">
+        <calcite-action data-action-id="layer-effects" icon="effects" text="Layer effects"></calcite-action>
```

Comment:
For UI cleanup, WDYT if we move the `arcgis-legend` into an `action` + `panel` combo?

```suggestion
        <calcite-action data-action-id="legend" icon="legend" text="Legend"></calcite-action>
        <calcite-action data-action-id="layer-effects" icon="effects" text="Layer effects"></calcite-action>
```